test/fixtures/minimum_duration_lapse_shipments.json

```diff
@@ -50,13 +50,6 @@
           "lon": -0.9744
         }
       },
-      {
```

Comment:
don't we need a redump for this file?

Reply:
We have just removed some points and replaced them by their doppelganger. Do you see any particular reason to redump the file ?

Reply:
The load of matrix dump also contain the association of the locations to the matrix indices so I see no issue here :wink: